src/python/grapl-tests-common/grapl_tests_common/setup_tests.py

```diff
@@ -13,6 +13,7 @@
 DUMP_ARTIFACTS = bool(environ.get("DUMP_ARTIFACTS", False))
 
 LOGGER = get_module_grapl_logger()
+GRAPL_LOG_LEVEL = os.getenv("GRAPL_LOG_LEVEL", "INFO")
```

Comment:
?

Reply:
Is there a question? I don't understand.

Reply:
oh duh you do actually use this, okay